nanomq/rest_api.c

```diff
@@ -11,8 +11,8 @@
 #include "include/broker.h"
 #include "include/nanomq.h"
 #include "include/sub_handler.h"
-#include "libs/base64.h"
-#include "libs/cJSON.h"
+#include <base64.h>
+#include <cJSON.h>
```

Comment:
?? why do you remove these header files?

Reply:
I've moved them to `nanolib`.

Reply:
So that I can call their functions in `nanolib` and `nanomq`.